### PR TITLE
Readme Update

### PR DESCRIPTION
**Pull request by aaron-bray**
I added a walk through of pulling, configuring, and building fletch for *nix and windows

I also updated the Getting Help as it was the same as the kwiver one, and i figured that was a copy/paste error.

### CODE REVIEW

**Comment by aaron-bray** [comment]
Here is the URL for my changes if it is easier to read

https://github.com/aaron-bray/fletch/tree/readme

**Comment by dstoup** [comment]
My only critique so far is the reference to build\release build\debug. That reminded me that we have issues on Windows with directories named release and debug when building Qt. I will patch the Qt build today, but we should rename the references to avoid confusion, or at least mention the bug for Windows users.

**Comment by aaron-bray** [comment]
Yeah, good point, I will note the bug, and change to 'rel' and 'deb'

File: README.rst [hunk 1]
```diff
@@ -124,18 +107,62 @@ want the C++ libraries built.
...
+  # Run CMake (it will use the system default compiler if you don't provide options or use the CMake GUI)
+  # Also, if using visual studio, you do no need to provide the build type
+  cmake -DCMAKE_BUILD_TYPE=Release -Dfletch_ENABLE_ALL_PACKAGES=ON -Dfletch_ENABLE_PYTHON=ON ../../src
+  # Again, python very popular option, but is optional
+  
+On Linux/OSX/MinGW, execute make
+  
+For MSVC users, open the generated fletch.sln and build the project in the configuration associated with the build folder.
+Even though MSVC supports building multiple configurations, you should only build one configuration per build folder.
+If you need multiple configurations you should create multiple subfolders and repeat the above insturctions for each configuration.
```
Comment (mleotta) [inline README.rst]: typo: insturctions --> instructions

**Comment by mleotta** [comment]
Looks good to me, modulo the typo

**Comment by aaron-bray** [comment]
Fixed both issues

**Comment by dstoup** [comment]
LGTM...merging

**Pull request merged**